assets/js/modules/analytics/datastore/profiles.js

```diff
@@ -146,25 +188,33 @@ export const actions = {
 		};
 	},
 
-	receiveProfilesCompleted( accountID, propertyID ) {
-		invariant( accountID, 'accountID is required.' );
-		invariant( propertyID, 'propertyID is required.' );
+	/**
+	 * Sets a profile based on given account and property IDs.
+	 *
+	 * @since n.e.x.t
+	 * @private
+	 *
+	 * @param {string} propertyID Property ID.
+	 */
+	*setProfileForProperty( propertyID ) {
```

Comment:
I agree that this is a bit inconsistent and most of this should probably go in `properties.js`. 

However, I'm not sure there would be a reason then to have a `selectProfile` action since this would essentially be `setProfileID`.